xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAmlogic.cpp

```diff
@@ -216,6 +216,10 @@ bool CDVDVideoCodecAmlogic::Open(CDVDStreamInfo &hints, CDVDCodecOptions &option
     }
   }
 
+  m_processInfo.SetVideoDecoderName(m_pFormatName, true);
+  m_processInfo.SetVideoDimensions(m_hints.width, m_hints.height);
```

Comment:
the definitions of processInfo's video dimensions is the uncropped size, the cropped size is already exposed by stream info. not sure if AML exposes this info. let me know if you want to change this or merge as is.